doc/usecases/core/UC_keyname_hierarchy.md

```diff
@@ -0,0 +1,29 @@
+# Use Case: `Key` Hierarchy
+
+## Summary
+
+- **Title:** Create `Key`
+- **Scope:** `libelektra-core`
+- **Level:** Developer Goal
+- **Actors:** Core, Caller
+- **Brief:** Hierarchy comparison between `Key`'s
```

Comment:
Please give name of function, otherwise one needs to guess what this is about.

Reply:
Once again, I intentionally avoid function names, because it is an implementation detail or if you want, it is on "decision level" not "use case level". The function name doesn't change anything about the use case. Different bindings will use different function names.